### PR TITLE
Newest readme

### PR DESCRIPTION
**Pull request by rachelzhang0922**
**What this PR does / why we need it**:

**Will this PR make the community happier**? 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**How is this PR tested**
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: README-zh.md [hunk 1]
```diff
@@ -12,63 +12,71 @@
...
+<div align="center">
 
-|特点|描述  |
-|---|---|
-|🔌 极速设备接入 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|可兼容各类协议及设备|
-|👨‍💻 高效应用开发|Shifu将每一个设备进行结构化虚拟，并将其能力以API的形式开放出来|
-|👨‍🔧 超低运维成本|Shifu使用Kubernetes原生框架，您无需再构建额外的运维基础设施|
+<h1 style="border-bottom: none">
+<br>
+    谷歌云物联网核心的开源替代
```
Comment (saiyan86) [inline README-zh.md:20]: 这个直接用英文就行，Google IoT Core的开源替代

File: README-zh.md [hunk 2]
```diff
@@ -12,63 +12,71 @@
...
+    <br />
+</h1>
+Shifu是一个k8s原生的, 生产级的,支持多协议设备接入的，开放的物联网开发平台。
```
Comment (saiyan86) [inline README-zh.md:23]: Shifu是一个k8s原生的， 生产级的，支持多协议设备接入的，开放物联网开发平台。

File: README.md [hunk 3]
```diff
@@ -9,53 +9,51 @@
...
+    <a href="https://github.com/Edgenesis/shifu"><img src="https://img.shields.io/github/stars/Edgenesis/shifu?style=social" height=25></a>
 </div>
-<br/><br/>
 
-# 🔧 Install
+# Features
+**Kubernetes native** — developing your application while managing your devices, spare the need for maintaining an additional O&M infrastructure
 
-- If you have a running Kubernetes cluster: Please use the command `kubectl apply` to install Shifu in your cluster:
+**Open Platform**— avoide vendor lock-in, you can easily deploy Shifu on public, private and hybrid cloud.
```
Comment (saiyan86) [inline README.md:41]: **Open Platform**— No vendor lock-in. You can easily deploy Shifu on the edge(from RaspberryPi to edge clusters) or on the cloud(public, private and hybrid cloud are all supported).

File: README.md [hunk 4]
```diff
@@ -9,53 +9,51 @@
...
 
-- If you don't have a running Kubernetes cluster: Please follow the following steps to try our demo.
-  - Download and Install Docker
-  
-    [Mac](https://docs.docker.com/desktop/install/mac-install/) | [Windows(WSL)](https://docs.docker.com/desktop/install/windows-install/) | [Linux](https://docs.docker.com/desktop/install/linux-install/)
-  - Download and Install Shifu Demo with a single command
-    ```sh
-    curl -sfL https://raw.githubusercontent.com/Edgenesis/shifu/main/test/scripts/shifu-demo-install.sh | sudo sh -
-    ```
+# How to connect a private-protocol camera with five lines of code
```
Comment (saiyan86) [inline README.md:48]: proprietary-protocol

**Review by saiyan86 — CHANGES_REQUESTED** [review]

File: README-zh.md [hunk 5]
```diff
@@ -12,63 +12,71 @@
...
 
+# Github Star数量
 
-# 🌟 GitHub Star 数量
 [![Stargazers over time](https://starchart.cc/Edgenesis/shifu.svg)](https://starchart.cc/Edgenesis/shifu)
+
 # 许可证
-This project is Apache License 2.0.
\ No newline at end of file
+This project is distributed under Apache 2.0 License.
```
Comment (kris21he) [inline README-zh.md:82]: translate to chinese?

Comment (rachelzhang0922) [reply README-zh.md:82]: 该项目使用Apache2.0许可证，is that the correct translation?

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@rachelzhang0922 need to remove submodule

**Review by tomqin93 — APPROVED** [review]

**Pull request merged**